scripts/libc-test.sh

```
@@ -174,12 +188,11 @@ echo "========================================"
 echo "  libc-test results: $PASSED/$TOTAL passed ($PCT%)"
 echo "========================================"
 
-if [ "$FAILED" -gt 0 ]; then
+if [ -n "$FAIL_LIST" ]; then
   echo ""
-  echo "Failed tests:"
-  printf "$ERRORS"
+  echo "Failed/hung tests:"
+  printf "$FAIL_LIST"
```

Shellcheck (0.11.0)

[info] Don't use variables in the printf format string. Use printf '..%s..' "$foo".

(SC2059)